examples/next-js-netlify/Earthfile

```diff
@@ -0,0 +1,38 @@
+VERSION 0.6
+FROM --platform=linux/amd64 node:latest
+WORKDIR /app
+
+deps:
+    ARG auth_token
+    ARG site_id
+    # Set environment variables required for building and deploying to Netlify
+    ENV NETLIFY_AUTH_TOKEN=$auth_token
+    ENV NETLIFY_SITE_ID=$site_id
+    # Install the Netlify CLI (global)
+    RUN npm install netlify-cli -g
+    # Copy package.json for installing required packages
+    COPY package.json ./
+    # Install the Netlify CLI (local) and Netlify Next.js plugin and required packages
+    RUN npm install netlify-cli --save-dev
+    RUN npm install @netlify/plugin-nextjs --save-dev
```

Comment:
Wondering if these can be saved in `package.json`, and then installed via `npm install --include dev`.

Reply:
They could be, but they aren't necessary packages for the site itself, only for using the Netlify CLI to build and deploy. If you build outside of the Netlify CLI or let Netlify's CI pick up the changes in your repo, you don't have to have these in `package.json`. So including them in the Earthfile makes more sense to me.